### DISCUSSION TITLE
Can I override a hard-code value of deployment.yaml during install Helm Chart?

### DISCUSSION THREAD

**Discussion by ma-hua** [fluxcd/flux2#3923] [Q&A]
Hi all! 

I got a helm chart that created by another team. 

But they used a hard-coded version of image in the deployment file for helm templates. So I can't override the image version by set values. Currently, I use helm-operator in my cluster and deploy helm charts via HelmRelease file.

Is there any way to replace this value? Any suggestions would be appreciated!

**Comment by hiddeco** [comment]
This is possible using a Kustomize post renderer, as long as it is not part of a hook. See https://fluxcd.io/flux/components/helm/helmreleases/#post-renderers for more information.

**Reply by ma-hua** [reply]
Thank you @hiddeco, this suit my requirements very well. And it is a sample of helm-controller.
Is there a similar solution in the helm-operator scenario, by editing the helm release file?
I can't migrate helm-operator to helm-controller yet. >_<
